sway-lib-std/src/lib.sw

```diff
@@ -45,3 +45,4 @@ pub mod low_level_call;
 pub mod array_conversions;
 pub mod bytes_conversions;
 pub mod clone;
+pub mod totalord;
```

Comment:
One more thing.
The trait and impl should be in `core::ops` instead of `std::totalord`